commons/clever-cloud-overview/common-application-configuration.md

```diff
@@ -70,3 +70,23 @@ repository. The script file **MUST** have execution right set.
 
 In the example above, the script is a file named `myScript` in the `clevercloud` folder.
 That is, right beside the `<instance_type>.json` file.
+
+## Workers
+
```

Comment:
Maybe it would be nice to explain that it's useful especially for PHP applications where you can't have long-running processes